src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -174,6 +177,8 @@ export const useSwapper = () => {
     state: { wallet },
   } = useWallet()
 
+  const flags = store.getState().preferences.featureFlags
```

Comment:
https://github.com/shapeshift/web/blob/simple-logger/src/hooks/useFeatureFlag/useFeatureFlag.tsx#L5

if you want, not a blocker

Reply:
Indeed: https://github.com/shapeshift/web/pull/2370/commits/c05b1e9df2d20cbad355961a385a636bc3b12c83